src/urllib3/util/retry.py

```diff
@@ -26,49 +25,7 @@
 )
 
 
-# TODO: In v2 we can remove this sentinel and metaclass with deprecated options.
-_Default = object()
-
-
-class _RetryMeta(type):
-    @property
-    def DEFAULT_METHOD_WHITELIST(cls):
-        warnings.warn(
-            "Using 'Retry.DEFAULT_METHOD_WHITELIST' is deprecated and "
-            "will be removed in v2.0. Use 'Retry.DEFAULT_METHODS_ALLOWED' instead",
-            DeprecationWarning,
-        )
```

Comment:
FYI, this deprecation warning hadn’t been visible, because it was emitted without a [`stacklevel`](https://docs.python.org/3/library/warnings.html#warnings.warn) and therefore [filtered by default](https://docs.python.org/3/library/warnings.html#default-warning-filter).

Reply:
@andersk I'm sorry that you haven't seen the warning, but I don't think stacklevel has anything to do about it? It just changes the way the stack trace is displayed

Reply:
It changes where Python thinks the warning originated, which changes whether it will be filtered. For example, this only shows a deprecation warning for `g`:

```console
$ cat a.py
import b
b.f()
b.g()

$ cat b.py
import warnings
def f():
    warnings.warn("f", DeprecationWarning)
def g():
    warnings.warn("g", DeprecationWarning, stacklevel=2)

$ python3 a.py
/tmp/a.py:3: DeprecationWarning: g
  b.g()
```

Reply:
Oh, wow, thanks for taking the time to educate us, I had not realized "defeating the purpose" meant "you won't see it". Will keep this in mind for any future warnings we add!

Sorry about that